tests/src/test-controllers/MethodTestsController.ts

```diff
@@ -1,9 +1,19 @@
+import { IsNumber, IsOptional, IsString } from 'class-validator';
 import { injectable } from "inversify"
 
-import { apiController, body, Controller, JsonPatch, rawBody, GET, POST, PUT, PATCH, DELETE } from "../../../dist/ts-lambda-api"
+import { apiController, body, Controller, JsonPatch, rawBody, GET, POST, PUT, PATCH, DELETE, bodyTyped } from "../../../dist/ts-lambda-api"
 
 import { Person } from "../test-components/model/Person"
 
+class TypedRequest {
```

Comment:
Could this be pulled out into a separate file?

Reply:
Will do